packages/es-components/src/components/navigation/sidenav/__snapshots__/SideNav.specs.js.snap

```diff
@@ -1,99 +1,234 @@
-      >
-        <span
-          className="sc-bxivhb khxFsg"
+          <a
+            class="home sc-bwzfXH igXrdK"
+            href="/home"
+          >
+            <span
+              class="sc-bxivhb khxFsg"
+            >
+              Home
+            </span>
+            <i
+              aria-hidden="true"
+              class="es-icon sc-bdVaJa clAcVq"
+              content="61f"
+              font-family="indv-mkt-icons"
+              font-size="inherit"
+            />
+          </a>
+        </li>
+        <li
+          class="es-sidenav__navitem"
         >
-          External Link
-        </span>
-        <i
-          aria-hidden={true}
-          className="es-icon sc-bdVaJa clAcVq"
-          content="61f"
-          fontFamily="indv-mkt-icons"
-          fontSize="inherit"
-        />
-      </a>
-    </li>
-    <li
-      className="es-sidenav__navitem"
-    >
-      <a
-        className="info sc-bwzfXH hJluX"
-        href="#/"
-        onClick={[Function]}
-      >
-        <span
-          className="sc-bxivhb khxFsg"
+          <a
+            class="cart sc-bwzfXH igXrdK"
+            href="#/"
+          >
+            <span
+              class="sc-bxivhb khxFsg"
+            >
+              Cart
+            </span>
+            <i
+              aria-hidden="true"
+              class="es-icon sc-bdVaJa clAcVq"
+              content="61f"
+              font-family="indv-mkt-icons"
+              font-size="inherit"
+            />
+          </a>
+        </li>
+        <li
+          class="es-sidenav__navitem"
+        >
+          <a
+            class="external sc-bwzfXH igXrdK"
+            href="http://www.google.com"
+            target="_blank"
+          >
+            <span
+              class="sc-bxivhb khxFsg"
+            >
+              External Link
+            </span>
+            <i
+              aria-hidden="true"
+              class="es-icon sc-bdVaJa clAcVq"
+              content="61f"
+              font-family="indv-mkt-icons"
+              font-size="inherit"
+            />
+          </a>
+        </li>
+        <li
+          class="es-sidenav__navitem"
         >
-          Disabled
-        </span>
-        <i
-          aria-hidden={true}
-          className="es-icon sc-bdVaJa clAcVq"
-          content="61f"
-          fontFamily="indv-mkt-icons"
-          fontSize="inherit"
-        />
-      </a>
-    </li>
-  </ul>
-</nav>
+          <a
+            class="info sc-bwzfXH hJluX"
+            href="#/"
+          >
+            <span
+              class="sc-bxivhb khxFsg"
+            >
+              Disabled
+            </span>
+            <i
+              aria-hidden="true"
+              class="es-icon sc-bdVaJa clAcVq"
+              content="61f"
+              font-family="indv-mkt-icons"
+              font-size="inherit"
+            />
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </div>,
+  "debug": [Function],
```

Comment:
It looks like this snapshot is spitting out the wrapped? component rather than the normal rendered object.

Reply:
oops. I'll fix this!